src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -482,6 +482,25 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
           </Box>
         )}
 
+        {/*Show any workflow-level errors at the top of the workflow details page.*/}
+        {workflow.selectedResult.exec_state.error !== null && (
+          <Box
+            sx={{
+              backgroundColor: theme.palette.red[100],
+              color: theme.palette.red[600],
+              p: 2,
+              paddingBottom: '16px',
+              paddingTop: '16px',
+              height: 'fit-content',
+              width: !drawerIsOpen ? `calc(100% - 148px)` : 'calc(100% - 32px)',
```

Comment:
What are these numbers coming from? Presumably they should be `const`s somewhere if they're not already defined?

Reply:
I think they're hardcoded in this file. Can comment with a more thorough explanation.